src/appleseed/renderer/modeling/bsdf/disneybrdf.cpp

```diff
@@ -451,7 +451,7 @@ namespace
                     sample);
                 sample.m_probability *= weights[SpecularComponent];
             }
-            else
+            else if (s < cdf[CleatcoatComponent])
```

Comment:
Is this test necessary? Isn't `cdf[CleatcoatComponent]` always 1.0, by definition of a CDF?